index.bs

```diff
@@ -116,12 +116,15 @@ Framework {#framework}
     // ...
     // More things from https://github.com/cure53/DOMPurify/blob/master/src/purify.js#L224
   };
+  typedef [StringContext=TrustedHTML] DOMString HTMLString;
+  typedef (DOMString or HTMLString or DocumentFragment or Document) SanitizerInput;
 
   [Exposed=(Window)]
   interface Sanitizer {
     constructor(SanitizerConfig? config);
-    DOMString toString(DOMString input);
-    DocumentFragment toFragment(DOMString input);
+    DocumentFragment sanitize(optional SanitizerInput input);
+    DOMString sanitizeToString(optional SanitizerInput input);
+    HTMLString sanitizeToHTML(optional SanitizerInput input);
```

Comment:
HTMLString -> TrustedHTML

(This specifies a DOMString return value, with an additional Trusted Type check, which won't really do anything for a return value.)

Reply:
Nit:
I think `HTMLString sanitizeToHTML` will confuse a lot of folks into expecting a string.
Especially as DOMString is very jargon-y for developers. :confused: 
Could we go for `sanitizeToTrustedHTML`, instead?

Reply:
Done 😊